solutions/standard/README.md

```diff
@@ -0,0 +1,12 @@
+# IBM Cloud Container Registry
+
+This architecture creates IBM Container Registry namespaces and provides the ability to configure pull traffic limits and storage quotas, both in megabytes. It ensures efficient management of container image access by regulating the volume of data that can be pulled from the registry and by setting specific storage capacity limits for container images within each registry:
+
+- A resource group, if existing is not passed in.
+- A Container Registry namespace.
+- Option to upgrade to `Standard` plan.
+- Option to set pull traffic and storage quotas.
+
+![IBM Container Registry](../../reference-architecture/deployable-architecture-icr.svg)
```

Comment:
I see in the diagram you have split into "Public namespace" and "Private namespace". I'm not sure this makes sense? I don't think the user can even configure a namespace to be public. Thats an internal process.

Reply:
updated the diagram to 
<img width="758" alt="Screenshot 2024-10-29 at 2 33 59 PM" src="https://github.com/user-attachments/assets/d1d6dee1-d2e1-4682-ba82-a7754a254e73">